packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -1529,6 +2029,22 @@ export class FieldEditor {
             {strDeleteModalMessage}
           </span>
         </fw-modal>
+        <fw-modal
```

Comment:
Can we create a new component called "fb-dependent-bulk-choices.tsx" for this and move all the render and functionality inside that component please. Else, it will be difficult to maintain the code

Reply:
I will take this as a separate PR post this review.